### PR TITLE
fix: update WorkflowTrigger crd

### PR DESCRIPTION
**Pull request by cd1989**
**What this PR does / why we need it**:

Update for WorkflowTrigger

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @zhujian7 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/apis/cyclone/v1alpha1/workflow_trigger.go [hunk 1]
```diff
@@ -30,21 +30,21 @@ const (
 
 // WorkflowTriggerSpec defines workflow trigger definition.
 type WorkflowTriggerSpec struct {
-	// Reference of the Workflow to trigger
-	WorkflowRef *corev1.ObjectReference `json:"workflowRef"`
 	// Type of this trigger, Schedule or Webhook
 	Type TriggerType `json:"triggerType"`
-	// Whether this trigger is enabled, if set to true, no workflow will be triggered
-	Enabled bool `json:"enabled"`
```
Comment (zhujian7) [inline pkg/apis/cyclone/v1alpha1/workflow_trigger.go]: Could you please explain why remove this `Enabled` ?

Comment (cd1989) [reply pkg/apis/cyclone/v1alpha1/workflow_trigger.go]: Put to status section

Comment (zhujian7) [reply pkg/apis/cyclone/v1alpha1/workflow_trigger.go]: I think `Enabled` is a *config* of the trigger, not a *status*. So maybe put it in `spec` is better. WDYT?

**Pull request merged**